homeassistant/components/media_player/sonos.py

```diff
@@ -1008,64 +997,61 @@ def source(self):
 
         return self._source_name
 
-    @soco_error
+    @soco_error()
     def turn_off(self):
         """Turn off media player."""
         if self._support_stop:
             self.media_stop()
 
-    @soco_error
-    @soco_filter_upnperror(UPNP_ERRORS_TO_IGNORE)
+    @soco_error(UPNP_ERRORS_TO_IGNORE)
```

Comment:
Just a comment: the constant `UNPN_ERRORS_TO_IGNORE` probably has a too generic name since it apparently only applies to some operations.